Mathlib/CategoryTheory/Core.lean

```diff
@@ -160,6 +164,33 @@ lemma coreAssociator {E : Type u₃} [Category.{v₃} E] {E' : Type u₄} [Categ
 
 end Iso
 
+namespace Core
+
+variable {G : Type u₂} [Groupoid.{v₂} G]
+
+lemma functorToCore_comp_left {G' : Type u₃} [Groupoid.{v₃} G'] (H : G ⥤ C) (F : G' ⥤ G) :
```

Comment:
For this assertion (and the next two), could you only state this as an isomorphism? (It may be unclear that we actually need equalities of functors here, and in general I would argue that it is better to get the isomorphism first, and then deduce the equality if it is *absolutely* needed using `Functor.ext_of_iso`.)